riotfile.py

```diff
@@ -141,6 +141,10 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
                 "types-PyYAML": latest,
                 "types-setuptools": latest,
                 "types-six": latest,
+                "types-requests": latest,
```

Comment:
Any of these riotfile changes should be in their own PR, since they are potentially fixes for generic CI issues and not related to this change.

e.g. if we had to roll back this PR, then we'd be rolling back these fixes.